torchmetrics/__init__.py

```diff
@@ -62,7 +62,7 @@
     RetrievalPrecision,
     RetrievalRecall,
 )
-from torchmetrics.text import WER, BERTScore, BLEUScore, ROUGEScore  # noqa: E402
+from torchmetrics.text import WER, BLEUScore  # noqa: E402
```

Comment:
I think all text metrics are free now...

Reply:
Is the change then necessary anymore?

Reply:
I think that we would exclude only the Image metrics which has any 3rd party dependency...